spec/index.html

```diff
@@ -1363,8 +1363,8 @@ <h4>Dataset-UNION</h4>
             <p>where union between graphs is defined as set-union of triples in those graphs.</p>
           </div>
           <div class="note">
-            <p class="prefix"><strong>Note:</strong></p>Note that, in the following, whenever we write Dataset-UNION( <em>X</em> ) where <em>X</em> = {DS<sub>1</sub>,DS<sub>2</sub>,...
-            ,DS<sub>n</sub>} is a set of datasets, we understand this as a shorthand for Dataset-UNION(DS<sub>1</sub>, Dataset-UNION(DS<sub>2</sub>, ... , Dataset-UNION(DS<sub>n</sub>,{})...)).
+            <p class="prefix"><strong>Note:</strong></p>Note that, in the following, whenever we write Dataset-UNION ( <em>X</em> ) where <em>X</em> = { DS<sub>1</sub>, DS<sub>2</sub>, ...,
+            DS<sub>n</sub> } is a set of datasets, we understand this as a shorthand for Dataset-UNION ( DS<sub>1</sub>, Dataset-UNION ( DS<sub>2</sub>, ..., Dataset-UNION ( DS<sub>n</sub>, {} ) ... ) ).
```

Comment:
This shows why the spaces are bad here. The nested forms do not look nested. Proper layout and styling is needed especially for narrow screed display.